zproject/computed_settings.py

```diff
@@ -699,8 +698,9 @@ def zulip_path(path: str) -> str:
 SCIM_LOG_PATH = zulip_path("/var/log/zulip/scim.log")
 
 ZULIP_WORKER_TEST_FILE = zulip_path("/var/log/zulip/zulip-worker-test-file")
-ZULIP_UPDATE_ANNOUNCEMENTS_LOCK_FILE = zulip_path(
-    "/home/zulip/deployments/zulip_update_announcements.lock"
+
+LOCKFILE_DIRECTORY = (
+    "/srv/zulip-locks" if not DEVELOPMENT else os.path.join(os.path.join(DEPLOY_ROOT, "var"))
```

Comment:
Is the motivation for putting this is the root of `var/` to avoid having to create an extra directory? I would have thought a `var/locks` would have been more natural there.

Reply:
I think we could just have the new decorator do a `os.makedirs(os.path.dirname(lock_path), exist_ok=True)` sort of thing if the directory is a concern.

Reply:
Yeah, because `zulip_path` also flattens everything into `~/zulip/var`.  But sure, a subdirectory is fine.   If `manage` commands are run without `puppet` having been run, it fails either way (either because `/srv/zulip-locks` doesn't exist, or for failing to have bits to make `/srv/zulip-locks`).